flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/hangar.flt

```diff
@@ -99,7 +99,6 @@ Pump.9=True
 BatterySwitch=False
 StructuralDeiceSwitch=False
 PropDeiceSwitch=False
-Autobrakes=1
 StandbyVacuum=False
```

Comment:
Do we need to define `A32NX_AUTOBRAKES_ARMED_MODE` here?

Reply:
I have no idea where this is used: is it when looking at the plane in menu file?
As there are other A32NX_XXX vars here I can add autobrake for at least for consistency purpose?

Reply:
I believe this is used when "previewing" the plane in the hanger in the main menu.

Reply:
Ok, added it with autobrake off just so we don't wonder why it's not there :)